src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -136,13 +136,10 @@ FitsSchemaInputMapper::FitsSchemaInputMapper(daf::base::PropertyList &metadata,
         // Read slots saved using an old mechanism in as aliases, since the new slot mechanism delegates
         // slot definition to the AliasMap.
         static std::array<std::pair<std::string, std::string>, 7> oldSlotKeys = {
-                std::make_pair("PSF_FLUX", "slot_PsfFlux"),
-                std::make_pair("AP_FLUX", "slot_ApFlux"),
-                std::make_pair("INST_FLUX", "slot_InstFlux"),
-                std::make_pair("MODEL_FLUX", "slot_ModelFlux"),
-                std::make_pair("CALIB_FLUX", "slot_CalibFlux"),
-                std::make_pair("CENTROID", "slot_Centroid"),
-                std::make_pair("SHAPE", "slot_Shape")};
+                {std::make_pair("PSF_FLUX", "slot_PsfFlux"), std::make_pair("AP_FLUX", "slot_ApFlux"),
+                 std::make_pair("INST_FLUX", "slot_InstFlux"), std::make_pair("MODEL_FLUX", "slot_ModelFlux"),
+                 std::make_pair("CALIB_FLUX", "slot_CalibFlux"), std::make_pair("CENTROID", "slot_Centroid"),
+                 std::make_pair("SHAPE", "slot_Shape")}};
```

Comment:
I agree the braces are weird. That's why I ran clang-format on the file (I guessed the braces were clang-format's fault and I wanted to check that). That confirmed by suspicions, but I was surprised when some other part of the file got reformatted (for the better, I think, though that's not very relevant).

I am running 5.0.0 installed using Homebrew